roles/edpm_ovn/tasks/configure.yml

```diff
@@ -46,6 +203,13 @@
   ansible.builtin.shell: >
     ovs-vsctl set open . {% for key, value in edpm_ovn_ovs_other_config.items() %} other_config:{{ key }}={{ value }} {% endfor %}
 
+- name: Disable mlockall in ovs
+  when: edpm_ovs_dpdk_vhost_postcopy_support|bool
+  replace:
+    path: '/etc/sysconfig/openvswitch'
```

Comment:
should the new options be applied somehow? e.g. through service restart?..

Reply:
Created New PR: https://github.com/openstack-k8s-operators/edpm-ansible/pull/328